libs/src/api/entityManager.ts

```diff
@@ -53,25 +66,226 @@ export class EntityManager extends Base {
     return trackIds
   }
 
-  /**
-   * Playlist default response values
-   */
-  getDefaultPlaylistReponseValues(): PlaylistOperationResponse {
+  getCurrentUserId() {
+    const userId: number | null = this.userStateManager.getCurrentUserId()
+    if (!userId) {
+      throw new Error('Missing current user ID')
+    }
+    return userId
+  }
+
+  getDefaultEntityManagerResponseValues(): EntityManagerOperationResponse {
     return {
       blockHash: null,
       blockNumber: null,
       error: null
     }
   }
 
-  /**
-   * Create a playlist using updated data contracts flow
-   */
+  /** Social Features */
+
+  async followUser(
+    followeeUserId: number
+  ): Promise<EntityManagerOperationResponse> {
+    const responseValues: EntityManagerOperationResponse =
+      this.getDefaultEntityManagerResponseValues()
+    try {
+      return await this.manageEntity({
+        userId: this.getCurrentUserId(),
+        entityType: EntityType.USER,
+        entityId: followeeUserId,
+        action: Action.FOLLOW,
+        metadataMultihash: ''
+      })
+    } catch (e) {
+      const error = (e as Error).message
+      responseValues.error = error
+      return responseValues
+    }
+  }
+
+  async unfollowUser(
+    followeeUserId: number
+  ): Promise<EntityManagerOperationResponse> {
+    const responseValues: EntityManagerOperationResponse =
+      this.getDefaultEntityManagerResponseValues()
+    try {
+      return await this.manageEntity({
+        userId: this.getCurrentUserId(),
+        entityType: EntityType.USER,
+        entityId: followeeUserId,
+        action: Action.UNFOLLOW,
+        metadataMultihash: ''
```

Comment:
would it be worthwhile to set the default value of metadataMultihash in the manageEntity method to default to an empty string? I could go either way, bc it would be bad to forget it/make sense to explicitly set to empty